src/System.Composition.Hosting/dir.props

```diff
@@ -0,0 +1,8 @@
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\dir.props" />
+  <PropertyGroup>
+    <Version>1.0.31</Version>
```

Comment:
We haven't shipped it.  I decided not to increment it because it's following the versioning scheme established previously in roxel and its unclear if it will ship on the same cadence as the rest of the product or be more one-off like Reflection.Metadata.  /cc @mellinoe